### ISSUE TITLE
More formatting options for text box component.

### ISSUE THREAD

**Issue by cleverdog** [RevealBi/Reveal.Sdk#201]
Hi team,

When creating a new text box in the RevealBI dashboard, there seem to be very limited options for formatting the text.  

![image](https://github.com/RevealBi/Reveal.Sdk/assets/5829221/5b547514-293f-4241-8284-85b66827dcb5)

Therefore, we would like to see more options like bolding, coloring, etc. Ideally, it would be perfect if there were options as good as the text editor in Slingshot.

![image](https://github.com/RevealBi/Reveal.Sdk/assets/5829221/17d8e2ae-8e61-46f1-9ca9-622d1e33fdc2)

**Comment by tharnotovmox** [comment]
i agree with this feature request, having the ability to set a background coloring will be helpful. Thanks
